meetings/2024/2024-04-30.md

```diff
@@ -0,0 +1,110 @@
+Mikhail from University of Bergen to talk about TC39-TG5 - Experiments in Programming Language Standardization
+* Travel fund update - Ben
+* Congrats to re-elected CPC Board members!
+
+### Board Meeting Updates
+
+- https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-board
+- Board votes will occur in two weeks (from prior board meeting) on items sent to board for approval
+- Note, there is discussion on ‘in kind’ membership, Ben and Robin to come forward with recommendations
+
+### Staff Updates
+
+- https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-legal-info
+
+https://github.com/openjs-foundation/cross-project-council/issues/1133 was tagged to CPC, and noted it may be related to Ecosystem and Sunsetting discussions
+
+- https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-staff-update
+- https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-website-update
+
+*Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
+
+### openjs-foundation/cross-project-council
+
+* Revisiting our policy to become a CPC member 
+[#1300](https://github.com/openjs-foundation/cross-project-council/issues/1300)
+Discussed during CPC call - how is access conflated with membership, and separating access from membership. Goal - propose we don’t have high barriers to becoming a CPC regular member, but that we ensure any additional access is always vetted. Refer to discussion/comments in the issue.
+Pruning inactive members is valuable
+Should we continue to block new applications for CPC regular membership? Explore this further in a working session? Joining the CPC does not automatically grant access, other than permission to attend private sessions. Does this mean having access to the private list?
+Being inclusive is important and valuable - avoid over-steering to avoid potential risks that are always part of standard operations
+Tagged for CPC working session
+
+* Selection of new Code of Conduct Team Community Members [#1298](https://github.com/openjs-foundation/cross-project-council/issues/1298)
+
+* Document CoC Team selection process [#1297](https://github.com/openjs-foundation/cross-project-council/issues/1297)
+
+* Revisit CPC meeting times [#1289](https://github.com/openjs-foundation/cross-project-council/issues/1289)
+
+* Election: 2024 CPC Directors [#1285](https://github.com/openjs-foundation/cross-project-council/issues/1285)
+
+* Create new Ecosystem Sustainability Program [#1277](https://github.com/openjs-foundation/cross-project-council/issues/1277)
+Board voted on this - unanimous approval, meetings are upcoming. No action needed from the CPC at this time, will keep the issue open until the meeting this week then it may be closed (bsternthal@linuxfoundation.org) - this is part of the group of items to be voted on by the board in two weeks.
```

Comment:
```suggestion
The board voted on this - unanimous approval, meetings are upcoming. No action is needed from the CPC at this time, will keep the issue open until the meeting this week then it may be closed (bsternthal@linuxfoundation.org) - this is part of the group of items to be voted on by the board in two weeks.
```